# Artificial-Intelligence-In-Bioinformatics-P3
I answered the following questions:

Question 1:
Explore chromosome X in Homo sapiens and find out the location of the MECP2 gene.
A. Which database did you use explore chromosome X from the Homo sapiens species and 
display a screenshot of chromosome X region?

B. Find MECP2 gene in the chromosome X and provide a screenshot.

C. What is the location of the MECP2 gene in the chromosomal assembly (ex: q32.1)?

D. Display the different coding regions of MECP2 gene and provide a screenshot?

E. Explore MECP2 protein coding gene and provide a screenshot. How many exons does this 
gene transcript have?

F. Locate the SNPs in the MECP2 gene and provide screenshots.


Question 2: 
Explore the information you extracted from your allocated chromosome and gene.
A. Explore the protein sequence that corresponds to MECP2 in UniProt and provide a 
screenshot of the UniProt entry.
• Hint: Can be obtained by exploring the transcript table displayed in Q1 section E.

B. What is the function of the protein?

C. Predict the secondary structure elements of this protein. Which tool did you use? Provide 
screenshots along with a brief analysis.

Question 3: 
Paste your protein sequence into https://swissmodel.expasy.org/interactive. Answer the follow 
questions (In your own words):
A. Which protein from the sequence has the most coverage?

B. What percent is the sequence’s identity.

C. Please attach a screenshot of the Model’s sequence alignment, the template, and the model 
generated as .pdb file.

D. Explore the 3D structure of the protein and provide the following details. 
i. How many helix and sheets and turns are in the structure? Why do we need to know 
how many helix and sheets? Please provide a screenshot of the structure. 
ii. Display the hydrogen bonds in the structure. Why do we want to display hydrogen 
bonds? Please provide a screenshot of the structure.

E. Discuss on the reliability of the model?


Question 4: 
Applying the information, you have extracted from the previous questions.
A. Find the protein sequence with highest similarity to that of your query sequence.

B. Which tool did you use and provide the screenshot of your result page?

C. Download the protein sequence with highest similarity (exclude Homo sapiens) and discuss 
the sequence.

D. Generate a 3D model for the similar sequence using Swiss Model.

E. Compare it with the Homo sapiens protein structure generated in Question 3, what do you 
observe by comparing two structures?

Question 5: 
Sometimes your tools do not work, sometimes the tools documentation is difficult to follow, and 
sometimes you need to reinvent the wheel. Your fellow college wrote a translation program. Their 
goal is to mimic the process of a DNA sequence converted to a mRNA sequence and then to a 
polypeptide (a sequence of amino acids). They do not have any biological background and need your 
help. They do not know how to program in Python and how some bugs in their code, therefore they 
would like you to fix (debug) their code.
A. What is wrong with the code in Figure 1 and why is it wrong.

B. Rewrite the code and explain how your rewritten code works.

C. Explain how your rewritten code would translate the given sequence to a polypeptide.
